tools/labs/docker/docs/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN apt-get install -y fonts-noto-cjk
 RUN apt-get install -y latexmk
 RUN apt-get install -y librsvg2-bin
```

Comment:
You need to add your signed-off-by line. 

https://stackoverflow.com/questions/1962094/what-is-the-sign-off-feature-in-git-for

If your properly set your:

```
$ git config --global user.name "John Doe"
$ git config --global user.email johndoe@example.com
```

then simply running `git commit --amend -s` should add the `SOB` line to your commit.

Then just force push the patch on your remote branch and should update this PR.

Other than it should be OK

Reply:
Fixed!
Thanks.